.github/workflows/main.yml

```diff
@@ -50,9 +50,9 @@ jobs:
       max-parallel: 2
       matrix:
         include: [
-          {msystem: MINGW32, toolchain: mingw-w64-i686, version: x32, gimp: "gimp", gimptool: "gimptool-2.0" },
```

Comment:
It's a bit more complicated than just adding this back in. the configure/make will need to be a bit more complex to allow for gimp2, gimp3, or both. fourier.c will likely need more if/then/else as time goes by and gimp3 grows beyond gimp-3.0. We would also need to keep fourier.exe for gimp2 (for continuity) and fourier3.exe for gimp3 since they both reside in the same project at the same time, so they can't have the same file name in the same directory.

Reply:
Yes, that is why I suggest keeping both GIMP2 & GIMP3 for Windows binaries that do not use configure/make. It is already working, I don't see the need to remove these two lines.

Reply:
doable.